packages/ag-charts-community/src/chart/chart.ts

```diff
@@ -172,12 +172,10 @@ export abstract class Chart extends Observable implements AgChartInstance {
         newValue(value: HTMLElement) {
             if (this.destroyed) return;
 
-            value.setAttribute('data-ag-charts', '');
```

Comment:
We use this for the dark mode logic (plugins/ag-charts-generate-example-files/src/executors/generate/generator/utils/getDarkModeSnippet.ts)

Reply:
Yes, I've moved this onto the part of the DOM we actually manage and updated the dark-mode logic.